tests/unit/test_utils.py

```diff
@@ -297,10 +297,11 @@ def test_is_special_file_for_non_existant_file(self):
         self.assertFalse(OSUtils().is_special_file(non_existant_filename))
 
     def test_get_temp_filename(self):
```

Comment:
This test shouldn't need to be updated as the filename is well under the max of 255 (and shouldn't change). Let's add a new test where the filename is over 255 and assert that it's truncated to the known length there.